ipi/engine/normalmodes.py

```diff
@@ -94,8 +94,12 @@ def __init__(self, mode="rpmd", transform_method="fft", propagator="exact",
             freqs = []
         if open_paths is None:
             open_paths = []
+        if bosons is None:
+            bosons = []
         self.open_paths = np.asarray(open_paths, int)
+        self.bosons = np.asarray(bosons, int)
         dself = dd(self)
+        dself.nmts = depend_value(name='nmts', value=nmts)
```

Comment:
I am just flagging this line to follow it up. Need to be a bit careful.

Reply:
Hi, not sure I understand what the problem is. Anything you need me to do?